doc/user_doc/vic_app_dev/container_operations.md

```diff
@@ -100,43 +100,45 @@ For more information about volume operations with vSphere Integrated Containers
 | **Command** | **Docker Reference** | **Supported** |
 | --- | --- | --- |
 |`volume create`|[Create a volume](https://docs.docker.com/engine/reference/commandline/volume_create/)|Yes|
-|`volume inspect`|[Information about a volume](https://docs.docker.com/engine/reference/commandline/volume_inspect/)|Yes, use with docker compose.|
-|`volume ls`|[List volumes](https://docs.docker.com/engine/reference/commandline/volume_ls/)|Yes. `--filter` is not supported.|
+|`volume inspect`|[Information about a volume](https://docs.docker.com/engine/reference/commandline/volume_inspect/)|Yes. Use with docker compose.|
+|`volume ls`|[List volumes](https://docs.docker.com/engine/reference/commandline/volume_ls/)|Yes|
 |`volume rm`|[Remove or delete a volume](https://docs.docker.com/engine/reference/commandline/volume_rm/)|Yes|
 
 <a name="compose"></a>
 ## Docker Compose Commands ##
 
-vSphere Integrated Containers Engine 0.8 supports Docker Compose version 1.10.
+vSphere Integrated Containers Engine 0.9 supports Docker Compose version 1.9.0.
 
 For more information about using Docker Compose with vSphere Integrated Containers Engine, see [Creating a Containerized Application with vSphere Integrated Containers Engine](creating_containerized_app_with_vic.md).
 
+For information about Docker Compose file support, see [Supported Docker Compose File Options](docker_compose_file_options.md).
+
 | **Command** | **Docker Reference** | **Supported** |
 | --- | --- | --- |
 | `build`  | [Build or rebuild service](https://docs.docker.com/compose/reference/build/)  | No. Depends on `docker build`.|
-| `bundle`  | [Generate a Distributed Application Bundle (DAB) from the Compose file](https://docs.docker.com/compose/reference/bundle/)| No|
+| `bundle`  | [Generate a Distributed Application Bundle (DAB) from the Compose file](https://docs.docker.com/compose/reference/bundle/)| No |
 | `config`  | [Validate and view the compose file](https://docs.docker.com/compose/reference/config/)  | Yes  |
 | `create`  | [Create services](https://docs.docker.com/compose/reference/create/)  | Yes  |
 | `down`  | [Stop and remove containers, networks, images, and volumes](https://docs.docker.com/compose/reference/down/)  | Yes  |
-| `events`  |[Receive real time events from containers](https://docs.docker.com/compose/reference/events/)  | No. Depends on `docker events`.|
+| `events`  |[Receive real time events from containers](https://docs.docker.com/compose/reference/events/)  | Yes. Supports passive Docker events for containers and images. When you perform `docker compose stop`, regular Docker Compose reports the `kill > die > stop` events, however vSphere Integrated Containers Engine reports `die > stop`. Regular Docker Compose reports the name of the image for any container operation, however vSphere Integrated Containers Engine reports `image=sha256:<hashtag>`. Does not yet support events for volumes or networks.|
```

Comment:
Can we just say this here:

Yes. Supports passive Docker events for containers and images. Does not yet support events for volumes or networks.